spec/cc/engine/analyzers/ruby/main_spec.rb

```diff
@@ -57,6 +57,49 @@
     end
   end
 
+  describe "#calculate_points(issue)" do
+    let(:ruby) { CC::Engine::Analyzers::Ruby::Main.new(engine_config: engine_conf) }
+    let(:base_points) { CC::Engine::Analyzers::Ruby::Main::BASE_POINTS }
+    let(:points_per) { CC::Engine::Analyzers::Ruby::Main::POINTS_PER_OVERAGE }
+    let(:threshold) { CC::Engine::Analyzers::Ruby::Main::DEFAULT_MASS_THRESHOLD }
```

Comment:
You might want to wrap this whole spec with `module CC::Engine::Analyzers::Ruby`:

```ruby
module CC::Engine::Analyzers::Ruby
  describe Main, in_tmpdir: true do
    # ...

     describe "#calculate_points(issue)" do
        let(:ruby) { Main.new(engine_config: engine_conf) }
        let(:base_points) { Main::BASE_POINTS }
        let(:points_per) { Main::POINTS_PER_OVERAGE }
        let(:threshold) { Main::DEFAULT_MASS_THRESHOLD }
      end

      # ...
  end
end